lib/tasks/new.rake

```diff
@@ -0,0 +1,18 @@
+namespace :admin do
+  desc "creates admin"
```

Comment:
@jynti Let's add an example to this description of how to execute this rake task.